### ISSUE TITLE
Adapt action box to the new layout

### ISSUE THREAD

**Issue by jsperezg** [decidim/decidim-initiatives#121]
# This is a Feature Proposal

#### :tophat: Description
Adapt actions to the new convention:

Check how it looks here:
https://meta.decidim.barcelona/processes/roadmap/f/122/proposals/12153

**Comment by xabier** [comment]
Please @decidim/lot-mods could you help with this? also @decidim/lot-core this is part of your integration tasks. We need to have this ASAP working on decidim.barcelona @jsperezg can do part of it but it needs to be integrated soon. cheers.

**Comment by mrcasals** [comment]
@xabier what's this "Action box", and what new layout? Also, if this is our job why is @decidim/lot-mods mentioned?

Cheers.

**Comment by xabier** [comment]
@decidim/lot-mods is mentioned because they just included the new action box to proposals (Adhesions). The new action-box is this: 

![image](https://user-images.githubusercontent.com/537284/37664124-fa1a9b16-2c5a-11e8-9598-e17d5b903077.png)

and comes to substitute the old one:

![image](https://user-images.githubusercontent.com/537284/37664194-18158a0e-2c5b-11e8-9f5a-17c5a259da8a.png)

the old one doesn't have the comment counter and action button, nor the endorsement counter and action button. These two buttons, like follow, should be present on almost any component of the platform.

**Comment by xabier** [comment]
@mrcasals since @jsperezg is importing this to consultation questions I wanted you and @decidim/lot-mods to help him do it right: particularly with some recent bugs found on the endorsement feature

**Comment by tramuntanal** [comment]
@jsperezg you can post me any questions you have, no problem. I developed the "reivindicar" button.
/cc @xabier

**Comment by jsperezg** [comment]
I've been checking the code for adhesions and It's highly coupled to proposals. From my understanding the desirable solution would be to create a polymorphic model DecidimAdhesions that any feature could implement and extrac/refactor all this code from proposals to Core. 

However my scope regarding the dead lines and the work load for each team involved is limited so it would be nice to hear your thougts regarding this question.

What is your opinion @xabier @mrcasals @tramuntanal @josepjaume ?

**Comment by xabier** [comment]
You are totally right @jsperezg Endorsements should be polymorphic, and displayed on any component (like comments or follow). I cannot evaluate the cost of it. Now, it needs to be implemented for consultation questions, this is for sure. Any progress on refactoring is welcome, but the feature applied to consultation-questions is the critical thing right now.

**Comment by xabier** [comment]
hi, given the cost of this integration I think it is better to abort this issue now and PR to decidim/decidim

**Comment by jsperezg** [comment]
Understood  @xabier I will create issues for all the engines that we support: initiatives, sorititions and consultations in order to merge them in decidim/decidim

**Comment by xabier** [comment]
great! thanks
